### ISSUE TITLE
The loss converge but the accuracy always stay zero with optimizer adam.

### ISSUE THREAD

**Issue by tinggh** [clovaai/deep-text-recognition-benchmark#23]
I add some characters and fine tuned the pretrained model on my dataset with parameters: TPS 、ResNet、 BiLSTM 、Attn 、sensitive 、adam. The loss converged quickly but the accuracy always stays zero. 
However, the accuracy would grow when I set the optimizer Adadelta.  
Could you give me some advises?

**Comment by ku21fan** [comment]
Hello, 

I recommend checking 2 parts.
1. try to use different learning rate with adam, such as `--lr 0.001`. 
Because the default setting uses `--lr 1` with adadelta.
2. By default, `--sensitive` option uses the characters below. 
```
opt.character = 0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~
```
Thus, when you use additional character, should change this line also.
https://github.com/clovaai/deep-text-recognition-benchmark/blob/3953e51381679ce32faf54d0f05d74274d4143a7/train.py#L251

Best

**Comment by tinggh** [comment]
Thank you for your in time reply. I think my problem is the first one and I will try it.
